### PR TITLE
Update fossa workflow's action version

### PR DESCRIPTION
**Pull request by shubham1172**
Signed-off-by: Shubham Sharma <shubhash@microsoft.com>

# Description

Due to a recent regression in FOSSA action's main branch, we are moving to the last working version.

## Issue reference

Related to: https://github.com/dapr/dapr/issues/4523

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Extended the documentation

### CODE REVIEW

**Review by artursouza — APPROVED** [review]

**Comment by berndverst** [comment]
Please don't merge as an approver or STC member. I was actively restarting the tests to make sure they pass before merging (because I can't overwrite the tests)

**Pull request merged**
